tests/ui_tools/test_boxes.py

```diff
@@ -401,13 +451,13 @@ def test_generic_autocomplete_set_footer(
             ("@Hum", 1, "@**Human 1**"),
             ("@Huma", 1, "@**Human 1**"),
             ("@Human", 1, "@**Human 1**"),
-            ("@Human 1", 0, "@**Human 1**"),
+            ("@Human 1", 0, "@**Human 1**"),  # Space-containing prefix
             ("@_H", 1, "@_**Human 1**"),
             ("@_Hu", 1, "@_**Human 1**"),
             ("@_Hum", 1, "@_**Human 1**"),
             ("@_Huma", 1, "@_**Human 1**"),
             ("@_Human", 1, "@_**Human 1**"),
-            ("@_Human 1", 0, "@_**Human 1**"),
+            ("@_Human 1", 0, "@_**Human 1**"),  # Space-containing prefix
```

Comment:
Can we call these "space-containing text" instead, consistent with their parameter's name?